messaging/client.go

```diff
@@ -335,12 +339,12 @@ func (c *Client) do(method, path string, values url.Values, contentType string,
 }
 
 // Conn returns a connection to the broker for a given topic.
-func (c *Client) Conn(topicID uint64) *Conn {
+func (c *Client) Conn(topicID uint64, dataURL *url.URL) *Conn {
```

Comment:
Can `dataURL` be different across calls to `Conn`? If not, why is it a parameter in the function call? Should it be set when the `Client` object is created?